mypy/plugins/dataclasses.py

```diff
@@ -141,11 +142,19 @@ def transform(self) -> None:
         if (decorator_arguments['init'] and
                 ('__init__' not in info.names or info.names['__init__'].plugin_generated) and
                 attributes):
+
+            args = [attr.to_argument() for attr in attributes if attr.is_in_init
+                    and not self._is_kw_only_type(attr.type)]
+
+            if info.fallback_to_any:
+                args = [Argument(Var('args'), AnyType(TypeOfAny.explicit), None, ARG_STAR),
+                        Argument(Var('kwargs'), AnyType(TypeOfAny.explicit), None, ARG_STAR2),
```

Comment:
I didn't really want to give the variables names (args, kwargs) but I couldn't find a better way to do it without writing a lot of code to duplicate `add_method`

Reply:
What happens if the child dataclass has a field named `args` or `kwargs`?

Reply:
Still works fine because we completely overwrite the args to __init__ with these args. The original fields won't appear

Reply:
Found a workaround without using variable names. Using an empty var for both arguments won't work unless it is the same empty var, that way mypy won't throw an error.

The main difference is in the result of `reveal_type(A)` where A is a dataclass inheriting from any. When using args and kwargs it appeared as:

```
a.py:9: note: Revealed type is "def (*args: Any, **kwargs: Any) -> a.A"
```

and not it appears as

```
a.py:9: note: Revealed type is "def (*Any, **Any) -> a.A"
```